pkg/asset/asset.go

```diff
@@ -55,3 +56,39 @@ func PersistToFile(asset WritableAsset, directory string) error {
 	}
 	return nil
 }
+
+// DeleteAssetFromDisk removes all the files for asset from disk.
+func DeleteAssetFromDisk(asset WritableAsset, directory string) error {
```

Comment:
Can we make this the internal `deleteAssetFromDisk`?  You don't seem to use it outside this package:

```console
$ git describe origin/pr/547
v0.3.0-113-g6e402c1
$ git grep DeleteAssetFromDisk 6e402c1
6e402c1:pkg/asset/asset.go:// DeleteAssetFromDisk removes all the files for asset from disk.
6e402c1:pkg/asset/asset.go:func DeleteAssetFromDisk(asset WritableAsset, directory string) error {
6e402c1:pkg/asset/store.go:		if err := DeleteAssetFromDisk(wa, s.directory); err != nil {
6e402c1:pkg/asset/store.go:		if err := DeleteAssetFromDisk(asset, s.directory); err != nil {
```